motr/ut/idx_dix.c

```diff
@@ -531,15 +531,18 @@ static void ut_dix_record_ops(bool dist, uint32_t cr_get_flags,
 	m0_free0(&op);
 	m0_free0(&rcs);
 
-	/* Try to add recs again without OVERWRITE flag. */
+	/*
+	 * Try to add recs again without OVERWRITE flag
+	 * (will be added due to versioning).
```

Comment:
I would like this line be changed:
With versioning enabled, it will succeed to put Key/Val pairs again when the keys already exist.

Reply:
Fixed